test/support/pages/SettingsPage.js

```diff
@@ -15,6 +15,36 @@ define(function (require) {
   SettingsPage.prototype = {
     constructor: SettingsPage,
 
+    clickAdvancedTab: function () {
+      common.debug('in clickAdvancedTab');
+      return common.findTestSubject('settingsNav advanced').click();
+    },
+
+    setAdvancedSettings: function setAdvancedSettings(propertyName, propertyValue) {
+      var self = this;
+      return common.findTestSubject('advancedSetting&' + propertyName + ' editButton')
+      .click()
+      .then(function () {
+        return common.sleep(1000);
```

Comment:
I removed the sleep and the test still passes locally.  Must have been left over from debugging.  Gone now.